openmc/settings.py

```diff
@@ -1043,6 +1061,36 @@ def _create_dd_subelement(self):
             subelement = ET.SubElement(element, "count_interactions")
             subelement.text = str(self._dd_count_interactions).lower()
 
+    def _create_resonance_scattering_element(self):
+        if self.resonance_scattering is None: return
+
+        element = ET.SubElement(self._settings_file, "resonance_scattering")
+
+        # Create an iterable version of resonance_scattering
+        if isinstance(self.resonance_scattering, Iterable):
+            res = self.resonance_scattering
+        else:
+            res = [self.resonance_scattering]
+
+        for r in res:
```

Comment:
If you choose to make `resonance_scattering` an `Iterable` then this `for` loop can also be simplified by iterating over `self.resonance_scattering`.